wsrpc_aiohttp/websocket/common.py

```diff
@@ -290,7 +301,24 @@ async def handle_method(self, method, serial, args, kwargs):
             args = a
 
         func = partial(callee, *args, **kwargs)
-        result = await self._executor(func)
+        try:
+            result = await self._executor(func)
+            await self.ON_CALL_SUCCESS.call(
```

Comment:
Your receive exception from `await self.ON_CALL_SUCCESS.call(` but not from `await self._executor(func)`